src/shubhamG/Test4_Program2.java

```diff
@@ -0,0 +1,32 @@
+package shubhamG;
+
+import java.util.Arrays;
+
+public class Test4_Program2 {
+
+	int getCountOfTreesVisible(int[] arr) {
+		int count = 1;
+		int maxHeight = arr[0];
+		for (int i = 1; i < arr.length; i++) {
```

Comment:
good to see that you have started loop from index 1, as you have already to see first tree